zip-0227.rst

```diff
@@ -84,9 +84,9 @@ The ZSA Protocol adds the following three keys to the key components [#protocol-
 
 1. The issuance master key, denoted as :math:`\mathsf{sk}_{\mathsf{iss}}`, as the name suggests, is the master key that is used to derive the other two keys.
 
-2. The issuance authorizing key is the key that is used to sign the issuance transaction, and is denoted as :math:`\mathsf{isk}`. This key is used to authorize the issuance of a specific Asset Identifier, and is only used by the issuer.
+2. The issuance authorizing key is the key that is used to sign the issuance transaction, and is denoted as ``isk``. This key is used to authorize the issuance of a specific Asset Identifier, and is only used by the issuer.
```

Comment:
The convention used in the protocol spec and other ZIPs is that byte sequence encodings are in monospace/mathtt, but other values are in mathsf.

Reply:
We were discussing the convention (see [here](https://github.com/QED-it/zips/pull/20#issuecomment-1579125270)) since there were some inconsistencies in what I had earlier, and I suggested this since `ik` is byte encoded, and it felt cleaner to me to have both `isk` and `ik` with the same formatting. 

Thanks for pointing out the convention, we'll go through this again and make it consistent.

Reply:
This has been updated in [PR#30](https://github.com/QED-it/zips/pull/30) and merged in.